src/Common/IconCache.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Smuxi - Smart MUltipleXed Irc
+//
+// Copyright (c) 2015 Carlos Martín Nieto
+//
+// Full GPL License: <http://www.gnu.org/licenses/gpl.txt>
```

Comment:
Please change the license the MIT and I am fine to re-license my GPLed code you based on. copyright line with my name is missing here.